storage/rocksdb/build_rocksdb.cmake

```diff
@@ -480,16 +600,57 @@ int main() {
   unset(CMAKE_REQUIRED_FLAGS)
 endif()
 
-IF(CMAKE_VERSION VERSION_GREATER "2.8.10")
-  STRING(TIMESTAMP GIT_DATE_TIME "%Y-%m-%d %H:%M:%S")
-ENDIF()
+option(WITH_ASAN "build with ASAN" OFF)
```

Comment:
I'm not sure its necessary to repeat `option`.

Reply:
Indeed, the duplication is unnecessary.